core/src/test/java/org/jenkins/ui/symbol/SymbolTest.java

```diff
@@ -0,0 +1,121 @@
+package org.jenkins.ui.symbol;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class SymbolTest {
+
+    @Test
+    @DisplayName("Get symbol should build the symbol with given attributes")
+    void getSymbol() {
+        String symbol = Symbol.get(new SymbolRequest.Builder()
+                                           .withName("science")
+                                           .withTitle("Title")
+                                           .withTooltip("Tooltip")
+                                           .withClasses("class1 class2")
+                                           .withId("id")
+                                           .build()
+        );
+        assertThat(symbol, containsString("<span class=\"jenkins-visually-hidden\">Title</span>"));
+        assertThat(symbol, containsString("tooltip=\"Tooltip\""));
+        assertThat(symbol, containsString("class=\"class1 class2\""));
+        assertThat(symbol, containsString("id=\"id\""));
+    }
```

Comment:
In order to see if that code would help for https://github.com/jenkinsci/prism-api-plugin/pull/59 I am wondering where you are actually checking for the SVG content? Can't you add an assertion for the most important part as well?

Reply:
Should be okay now.